internal/ui/settings_ui.go

```diff
@@ -60,22 +63,29 @@ func (d *settingsUI) loadAppearanceScreen() fyne.CanvasObject {
 		bgPathClear.Disable()
 	}
 	bgLabel := widget.NewLabelWithStyle("Background", fyne.TextAlignCenter, fyne.TextStyle{Bold: true})
+	bgDialog := dialog.NewFileOpen(func(file fyne.URIReadCloser, err error) {
+		if err != nil || file == nil {
+			return
+		}
+
+		// not advisable for cross-platform but we are desktop only
+		path := file.URI().String()[7:]
+		// TODO add a nice preview :)
+		_ = file.Close()
+
+		bgPath.SetText(path)
+		bgPathClear.Enable()
+	}, d.win)
+	bgDialog.SetFilter(&storage.MimeTypeFileFilter{MimeTypes: []string{"image/jpg", "image/jpeg", "image/png", "image/svg"}})
```

Comment:
I think that the correct mime for SVG is "image/svg+xml".
However I would use the file extension filter personally - that is what you want to filter right?
If for some reason a system is missing the mime lookups this will not do what you intend.

Reply:
Good point. Will switch to extensions :)